applications/eradius_prometheus_collector/src/eradius_prometheus_collector.erl

```diff
@@ -20,12 +20,18 @@
 
                   {access_requests_total, counter, "Amount of Access requests requests received by the RADIUS server"},
                   {accounting_requests_total, counter, "Amount of Accounting requests requests received by RADIUS server"},
+                  {accounting_requests_start, counter, "Amount of Accounting requests requests with 'start' type of received by RADIUS server"},
```

Comment:
I think you can remove ''of'' from the description. Same for the other new counters.